# emr2_moveit_config

usage
$ ros2 launch emr2_moveit_config demo.launch.py


Bug-Solver

[ERROR] [launch]: Caught exception in launch (see debug for traceback): 'capabilities'

=> https://github.com/moveit/moveit2/issues/2734
